lib/order.rb

```diff
@@ -8,11 +10,58 @@ def initialize(id, products)
     end
 
     def total
-      # TODO: implement total
+      # returns order total with tax
+      sum = products.values.inject(0, :+)
+      total = (sum + (sum * 0.075)).round(2)
```

Comment:
Good use of `inject` here. For the tax, it might be a little cleaner to multiply by 1.075.